src/Hosting.Services/HostBuilderExtensions.cs

```diff
@@ -101,16 +103,20 @@ public static IServiceCollection AddOmexServiceFabricDependencies<TContext>(this
 						options.ValidateOnBuild = true;
 						options.ValidateScopes = true;
 					})
-					.ConfigureLogging(builder => builder.AddOmexLogging())
 					.Build();
 
+#pragma warning disable OMEX188 // InitializationLogger using OmexLogger is obsolete. DiagnosticId = "OMEX188"
 				InitializationLogger.LogInitializationSucceed(serviceNameForLogging);
```

Comment:
If AddOmexLogging is no longer called, how will this InitializationLogger work?

Reply:
I dont fully get your question. however InitializationLogger is a static class/method which is isolated from DI using AddOmexLogging